packages/core-ethereum/src/channel.ts

```diff
@@ -191,7 +191,7 @@ class Channel {
       UINT256.fromString('0'),
       new UINT256(new BN(this.index++)),
       new Balance(new BN(0)),
-      UINT256.fromProbability(1),
+      UINT256.fromInverseProbability(new BN(new Uint8Array(UINT256.SIZE).fill(0xff))),
```

Comment:
I would use `new BN(ethers.constants.MAX_UINT256)`

Reply:
This means that we need access to `ethers.js` whenever we want to issue dummy tickets, moving this into a static property

Reply:
Changed to a constant in `types/solidity.ts`